server/routes/index.js

```diff
@@ -26,7 +26,7 @@ r.get('/kaboom', (ctx, next) => {
 });
 r.get('/download', (ctx, next) => {
   const uri = ctx.request.query.uri;
-  if (uri.match('https://iiif.wellcomecollection.org')) {
+  if (uri.match('https://iiif.wellcomecollection.org') || uri.match('https://prismic-io.s3.amazonaws.com/wellcomecollection')) {
```

Comment:
```JavaScript
const allowedDomains = [
  'https://iiif.wellcomecollection.org',
  'https://prismic-io.s3.amazonaws.com/wellcomecollection'
];

if (uri.match(new RegExp(allowedDomains.join('|')))) { /* do it */ }
```

Because I haven't done any coding today, just a suggestion.

Reply:
oh nice! I thought there'd be a slick way of doing this.